test/integration/account_oauth_client_test.go

```diff
@@ -26,14 +28,14 @@ func TestGetOAuthClient_missing(t *testing.T) {
 	}
 }
 
-func TestGetOAuthClient_found(t *testing.T) {
+func TestOAuthClient_GetFound(t *testing.T) {
 	createOpts := linodego.OAuthClientCreateOptions{
 		Public:      true,
 		RedirectURI: "https://example.com",
-		Label:       "test-client",
+		Label:       fmt.Sprintf("test-client-%.d", time.Now().Second()),
```

Comment:
We might want to leave these the same since running tests against fixtures will return inconsistent values for `time.Now().Second()`. Normally this isn't a problem but it can cause issues when we compare against `Label` further down.